macros/edr/tests/test_utils/get_elementary_test_table.sql

```diff
@@ -0,0 +1,32 @@
+{% macro get_elementary_test_table(test_relations, test_name, suffix) %}
+    {% set test_table_name = elementary.table_name_with_suffix(test_name, suffix) %}
+    {% set relation_keys = adapter.dispatch('get_relation_keys', 'elementary')() %}
+    {% for test_relation in test_relations %}
```

Comment:
_Might_ need to optimize this. `test_relations` is an [`agate.Table`](https://agate.readthedocs.io/en/latest/api/table.html) object, it has `.find()` methods but I imagine it does roughly the same.